src/styles/global/components.scss

```diff
@@ -6,7 +6,7 @@
 
 // Common
 @import "src/components/common/Footer/footer.scss";
-@import "src/components/common/PostalCode/postal-code-input.scss";
+@import "src/components/common/PostalCode/postal-code.scss";
```

Comment:
same here, regard  `PostalCode` component be part of the Regionalization feature, it should be in Regionalization imports (below line 19)